mp/src/game/shared/momentum/mom_rocket.cpp

```diff
@@ -309,7 +370,14 @@ CMomRocket *CMomRocket::EmitRocket(const Vector &vecOrigin, const QAngle &vecAng
     pRocket->SetRadius(146.0f);
 
     pRocket->CreateSmokeTrail();
-    pRocket->EmitSound("Missile.Ignite");
+
+    static ConVarRef mom_rj_sounds("mom_rj_sounds");
+    const bool bMomSounds = !mom_rj_use_tf_rocketmodel.GetBool() && mom_rj_sounds.GetInt() != 0;
+    const bool bTF2Sounds = !mom_rj_use_tf_rocketmodel.GetBool() && mom_rj_sounds.GetInt() == 2;
```

Comment:
I don't understand this logic. It looks to me like `bMomSounds` will always be true when `bTF2Sounds` is true.